tools/tests/test_lint.py

```
@@ -252,6 +252,62 @@ def test_ledger_row_bad_vocab_values_are_findings(tmp_path):
+
+
+def test_ledger_judged_implementation_position_is_lawful(tmp_path):
+    _write_ledger(tmp_path, _ledger_row(introduced="implementation", catchable="implementation"))
+    assert lint.run(tmp_path) == []
+
+
+def test_ledger_unjudged_and_implementation_are_separate_values():
+    """Criterion: no value doubles as a judgment and a default. Goes red if the
+    not-judged value is ever collapsed onto a real position."""
+    assert {"unrecorded", "implementation"} <= lint.LEDGER_POSITIONS
+    assert "unrecorded" != "implementation"
```

Ruff (0.16.1)

[warning] Two constants compared in a comparison, consider replacing `"unrecorded" != "implementation"`

(PLR0133)